content/blog/2017/2017-04-10-Jenkins-has-upgraded-to-Java-8.adoc

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Jenkins Now Requires Java 8 To Run Starting With 2.54"
+tags:
+- java8
+- upgrade
+author: batmat
+---
+
+:imagesdir: /images/post-images/2017-04-java8
+
+image:lambda.png[role="right"] As link:/blog/2017/01/17/Jenkins-is-upgrading-to-Java-8/[announced some weeks ago], the necessary changes have now been merged into Jenkins codebase.
+That means that starting with link:https://jenkins.io/changelog/#v2.54[Jenkins **2.54**], the Java runtime required for Jenkins is updated to Java 8.
```

Comment:
"…the Java runtime requirement for Jenkins has been upgraded to Java 8"?  Or "updated" or "changed", rather than upgraded.